test/functional/legacy/062_tab_pages_spec.lua

```diff
@@ -139,6 +139,7 @@ describe('tab pages', function()
 	  autocmd TabLeave * :call add(g:r, 'TabLeave')
 	  autocmd WinLeave * :call add(g:r, 'WinLeave')
 	  autocmd BufLeave * :call add(g:r, 'BufLeave')
+	  autocmd TabNew   * :call add(g:r, 'TabNew')
```

Comment:
We have `test/functional/autocmd/tabnew_spec.lua` and TabNew is not part of legacy Vim, so it should really be tested there.